testing/src/test/scala/org/http4s/testing/ErrorReporting.scala

```diff
@@ -31,11 +55,11 @@ object ErrorReporting {
         result
       } catch {
         case NonFatal(e) =>
-          System.setErr(oldErr)
+//          System.setErr(oldErr)
           // In case of errors, print whatever was caught
-          fakeErr.close()
-          val out = outStream.toString("utf-8")
-          if (out.nonEmpty) oldErr.println(out)
+//          fakeErr.close()
+//          val out = outStream.toString("utf-8")
+//          if (out.nonEmpty) oldErr.println(out)
```

Comment:
We should leave this method as it was, as it's still useful for silencing things actually logged to `System.err`.